### DISCUSSION TITLE
eza v0.10.8

### DISCUSSION THREAD

**Discussion by cafkafk** [eza-community/eza#159] [Announcements]
# Description
This release marks the 100'th pull request, and passes the 2000'th commit! :tada: 

This is really cool. Thanks for all the contributions, issues, and just for using eza!

Notable changes include:
- `--blocksize` had been added/revamped, and is much better than before.
- When using `--icons`, the folders now indicate whether they're empty or full.
- There is now an installation option for Ubuntu/Debian users

For developers:
- We are working hard on an integration testing solution and better guide for working on eza, both of these efforts are soon done, so hang tight!

In general, I've thought that a release schedule should soon be in order, perhaps figuring this out will accompany a major version release. Perhaps not, we'll find out soon.

# Changelog

### Bug Fixes

- TextCell building of detailed grid view for  hyperlink and icon options
- Block's Colours trait as for file sizes
- --blocksize completion, new description
- Option.views unit tests use --blocksize
- Add missing colon before -w/--width
- Replace exa by eza in help string
- Change exa to eza in invalid option error
- Add missing name section to eza_colors-explanation manpage
- Replace exa by eza in .gitignore

### Documentation

- Update issue templates
- Add git-ignore style/color information to manpage
- --blocksize, new description
- --blocksize, new description
- --blocksize, new description
- Cafkafk -> eza-community
- Add gpg public key for the deb repository
- Add section about debian and ubuntu installation

### Features

- Add `just` and `pandoc` to devShell bc they are necessary for man
- Add `.envrc` so direnv automatically opens the nix dev environment
- Add git-ignored color/style option
- Match folder icon to reflect contents
- Match folder icon to reflect contents
- --blocksize completion, new description
- Add script deb-package.sh

### Miscellaneous Tasks

- Bump libc from 0.2.93 to 0.2.147
- Bump num_cpus from 1.13.0 to 1.16.0
- Bump git2 from 0.16.1 to 0.17.2
- Bump unicode-width from 0.1.8 to 0.1.10

### Refactor

- Fs::fields::Blocks
- File::blocks() name, revise calculation
- Rendering Blocksize like file sizes
- Rename Blocks column to Blocksize
- Use -S/--blocksize and, var BLOCKSIZE
- Unit tests for output.render.blocks
- Flip if (as suggested/demanded by clippy)

### Build

- Add charm to nix develop
- Add tests/tmp to gitignore
- Add initial tape
- Add test runner sketch
- Add test runner to justfile
- Add out.gif to .gitignore
- Add run_tests  NAME arg
- Add reference main.txt
- Add gen_test
- Fix typo
- Handle arbitrary NAMES
- Remove commented out code
- Fix code formatting
- Add vhs-runner main function
- Gen_test support automatic gen
- Automatic tape detection
- Add print_msg with ansi color
- Slight documentation/refactor
- Use ansi output on all output
- Disable vhs publish ad
- Add better tracing
- Remove defective sed
- Add color variables
- Add eza-long test
- Add itest testing dir
- Add parallel runner

### Ci

- Help text in xtests
- Nix flake check
- Add labeler for flake
- Add flake description

### Git

- Add deb package to .gitignore

## New Contributors
* @PThorpe92 made their first contribution in https://github.com/eza-community/eza/pull/133
* @sefidel made their first contribution in https://github.com/eza-community/eza/pull/141

**Full Changelog**: https://github.com/eza-community/eza/compare/v0.10.7...v0.10.8

<hr /><em>This discussion was created from the release <a href='https://github.com/eza-community/eza/releases/tag/v0.10.8'>eza v0.10.8</a>.</em>

**Comment by gierens** [comment]
Just updated the deb repo.
